### PR TITLE
Fix slider when no article

### LINKED ISSUE #4373: [BUG] Manage feed slider
**Describe the bug**
I guess it comes from ##4291

When there is no unread feed in the stream, than the "manage" link does not open the slider. It opens the manage feed in a new page

**To Reproduce**
Prerequisite: Not all feeds are included in the main stream.

1. Go to normal view, go to main stream
2. mark all feeds as read
3. (The message "There are no articles to show." is shown)
4. open the sub menu of a feed
5. click on manage
6. the options are opened in a new page

grafik

grafik

**Expected behavior**
The slider is opened to edit the feed

**Environment information (please complete the following information):**
- FreshRSS version: 1.20.0-dev

### PR DESCRIPTION
**Pull request by Alkarex**
#fix https://github.com/FreshRSS/FreshRSS/issues/4373
https://github.com/FreshRSS/FreshRSS/pull/4291

### CODE REVIEW

File: app/views/index/normal.phtml [hunk 1]
```diff
@@ -138,4 +134,11 @@ $today = @strtotime('today');
 </main>
 <?php endif; ?>
 
+<?php $class = $this->displaySlider ? ' active' : ''; ?>
+<a href="#" id="close-slider" class="<?= $class ?>">
+	<?= _i('close') ?>
+</a>
+<div id="slider" class="scrollbar-thin<?= $class ?>">
```
Comment (math-GH) [inline app/views/index/normal.phtml]: I like the improvement, that it is outside of `<main>`
Would be `<aside>` the better semantic tag, while it is now outside of `<main>`?

**Review by math-GH — APPROVED** [review]
It works fine for me.

File: app/views/index/normal.phtml (hunk 1 above)
Comment (Alkarex) [reply app/views/index/normal.phtml]: Sounds like a good idea

Comment (Alkarex) [reply app/views/index/normal.phtml]: This required a few more changes https://github.com/FreshRSS/FreshRSS/pull/4375/commits/33b3605e473f3cc0032cf622b51df3e0858f2e41 https://github.com/FreshRSS/FreshRSS/pull/4375/commits/c4eeafc676d836cd2e754f1df06bfa5bf15697a6
Tests welcome

**Comment by Alkarex** [comment]
Ups, I had some unsaved files... https://github.com/FreshRSS/FreshRSS/pull/4375/commits/c4eeafc676d836cd2e754f1df06bfa5bf15697a6

**Comment by math-GH** [comment]
approved

**Review by math-GH — APPROVED** [review]

**Pull request merged**